src/code/z_actor.c

```diff
@@ -3484,13 +3459,12 @@ typedef struct {
     /* 0x18 */ u32 unk_18;
 } struct_801160DC; // size = 0x1C
 
-static struct_801160DC D_801160DC[] = {
-    { 0.54f, 6000.0f, 5000.0f, 1.0f, 0.0f, 0x050011F0, 0x05001100 },
-    { 0.644f, 12000.0f, 8000.0f, 1.0f, 0.0f, 0x06001530, 0x06001400 },
-    { 0.64000005f, 8500.0f, 8000.0f, 1.75f, 0.1f, 0x050011F0, 0x05001100 },
-};
-
 void Actor_DrawDoorLock(GlobalContext* globalCtx, s32 frame, s32 type) {
+    static struct_801160DC D_801160DC[] = {
+        { 0.54f, 6000.0f, 5000.0f, 1.0f, 0.0f, 0x050011F0, 0x05001100 },
+        { 0.644f, 12000.0f, 8000.0f, 1.0f, 0.0f, 0x06001530, 0x06001400 },
+        { 0.64000005f, 8500.0f, 8000.0f, 1.75f, 0.1f, 0x050011F0, 0x05001100 },
+    };
```

Comment:
Is there a reason to move all these variables inside functions? Imo we should avoid doing that unless it's needed or really makes sense (a lot of the things you moved make more sense outside to me)

Reply:
agreed, just cause something is only used in one function doesnt always mean it makes sense as in function static imo

Reply:
hm why avoid it when it seems to clearly be correct? I've been doing this on every file I work on thus far

Reply:
Adding on to this not having as much static might make it easier to find the bss issue.

Reply:
bss is already fixed, and everything in here should be static at the very least unless used globally

Reply:
Do we even know what is this array for?
I think we should not move those variables into of functions until we figure out their meaning.

Reply:
now we do :)